src/index.js

```diff
@@ -7,9 +7,17 @@ import 'react-widgets/dist/css/react-widgets.css';
 import 'babel-polyfill';
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { systemActions } from '@openmrs/react-components';
+import setupStoreAndPersistor from './store'
 import './index.css';
 import App from './App';
-import { unregister } from './registerServiceWorker';
+import register from './registerServiceWorker';
+
+
+const onUpdateSW = () => {
+  const { store } = setupStoreAndPersistor();
```

Comment:
@jomadoye why do we have to set up the store again here?

Reply:
@mogoodrich, I needed access to the current store so I could dispatch and action. thinking through, I see how this might be creating a new store, lemme take a look.

Reply:
I'm fine this this if it doesn't break anything or reset the store or cause problems... I didn't look at what setupStoreAndPersistor() does...

Reply:
@mogoodrich I just updated how the store is being setup. Please take a look.

Reply:
Looks good, thanks @jomadoye !